### ISSUE TITLE
Add quick links table to argparse docs

### ISSUE THREAD

**Issue by rhettinger** [python/cpython#65349]
BPO | [21150](https://bugs.python.org/issue21150)
--- | :---
Nosy | @rhettinger, @terryjreedy, @merwok, @lisroach, @mlouielu, @suhearsawho
PRs | <li>python/cpython#1210</li><li>python/cpython#12005</li>
Files | <li>[SampleOfSummaryTable.PNG](https://bugs.python.org/file45972/SampleOfSummaryTable.PNG "Uploaded as image/png at 2016-12-19.23:24:11 by Gaurav Tatke")</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2014-04-03.20:47:15.257>
labels = ['easy', 'docs']
title = 'Add quick links table to argparse docs'
updated_at = <Date 2019-02-24.03:03:06.257>
user = 'https://github.com/rhettinger'
```

bugs.python.org fields:
```python
activity = <Date 2019-02-24.03:03:06.257>
actor = 'suhearsawho'
assignee = 'docs@python'
closed = False
closed_date = None
closer = None
components = ['Documentation']
creation = <Date 2014-04-03.20:47:15.257>
creator = 'rhettinger'
dependencies = []
files = ['45972']
hgrepos = []
issue_num = 21150
keywords = ['patch', 'easy']
message_count = 8.0
messages = ['215468', '215568', '215858', '283653', '291887', '292296', '335492', '335495']
nosy_count = 10.0
nosy_names = ['rhettinger', 'terry.reedy', 'bethard', 'eric.araujo', 'docs@python', 'paul.j3', 'lisroach', 'Gaurav Tatke', 'louielu', 'suhearsawho']
pr_nums = ['1210', '12005']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue21150'
versions = ['Python 3.5']
```

</p></details>

**Comment by rhettinger** [comment]
Gaurav, overall this looks nice.  

I would drop the example column because it will make the row too wide for viewing.  Alternatively, put the examples in a separate table (as was done for the combinatoric functions in the itertools docs: https://docs.python.org/3/library/itertools.html#module-itertools ).

**Comment by 9da18a29-a07d-4828-a801-24cc911246de** [comment]
Could Raymond, Gaurav or paul help to review the PR's summary table?
Thanks!

**Comment by JulienPalard** [comment]
https://github.com/python/cpython/pull/12005 has been merged, it adds a [Quick Links for add_argument()](https://docs.python.org/dev/library/argparse.html#quick-links-for-add-argument)", for me it's enough to close this issue.

Does anyone think we need more than `add_argument` quicklinks?

**Comment by hauntsaninja** [comment]
Closing, since #12005 was merged and it looks like #91726 and #91762 addressed the open comments on #12005
